packages/default/src/controls/input.control.tsx

```diff
@@ -12,14 +13,15 @@ import {
 import { connect } from 'react-redux';
 
 export const InputControl =
-  ({ classNames, id, errors, label, uischema, schema, visible }: ControlProps) => {
+  ({ classNames, id, errors, label, uischema, schema, visible, required }: ControlProps) => {
   const isValid = errors.length === 0;
   const divClassNames = 'validation' + (isValid ? '' : ' validation_error');
+  const trim = uischema.options && uischema.options.trim;
 
   return (
-    <div className={classNames.wrapper} hidden={!visible}>
+    <div className={classNames.wrapper} hidden={!visible} data-trim={trim}>
```

Comment:
Can we re-use the styling registry here instead of setting a custom `data-trim` attribute?
On a different note: Shouldn't we move the initialization of the styling registry into the default set as well, just like with the material set?

Reply:
Yes, we could use the styling registry.
Yes we should move the init of the styling registry into default